modules/docs/markdown/04-codegen/01-customisation.md

```diff
@@ -270,6 +270,50 @@ Whether the provider is in the companion object or not, it must be `implicit`.
 
 :::
 
+#### Parameterised Types
+
+As of smithy4s version 0.17, you can now create refinements on types that take a generic type parameter. This can be accomplished by setting `parameterised` to `true` as seen below.
+
+```kotlin
```

Comment:
```suggestion
```smithy
```

Reply:
Not sure if our docusaurus site does highlighting for smithy yet